README.md

```diff
@@ -204,6 +204,24 @@ You can add or remove input and output variables at the stack level by following
 1.  Select a member configuration.
 1.  From the deployed details window, you can promote any of the configuration inputs or outputs.
 
+### Selective provisioning of observability resources
+
+You can selectively provision observability resources such as Activity Tracker routes and targets, and Cloud Monitoring instances by following these steps:
+
+1.  In the {{site.data.keyword.cloud_notm}} console, click the **Navigation menu** icon ![Navigation menu icon](/images/icon_hamburger.svg "Menu") > **Projects**.
+2.  Click the project with the stacked deployable architecture that you want to update.
+3.  Click the **Configurations** tab.
+4.  **Navigate to the Configurations**: Go to the Projects page and access the "Logging Monitoring Activity Tracker" settings under observability resources.
+5.  **Edit Configuration**: Use the Edit option to access the configuration page.
+6.  **Locate Optional Variables**: Under the Optional tab, you’ll find the specific variable settings.
+7.  From the deployed details window, you can enable or disable the provisioning of specific observability resources. For example:
+    - **IBM Cloud Logs instance** (`cloud_monitoring_provision`): Set this to provision or skip provisioning an IBM Cloud Monitoring instance.
```

Comment:
You mention Cloud Logs and Cloud Monitoring in this sentence. These should be separate bullet points - they are two different services that are toggled on or off using two different boolean variables